### PR TITLE
Add some Portal Unit Tests

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

**Review by mhuseinov — APPROVED** [review]
👍

File: portal-frontend/src/app/services/document/document.service.spec.ts [hunk 1]
```diff
@@ -23,4 +44,27 @@ describe('DocumentService', () => {
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  it('should make a get request for getUploadUrl', async () => {
+    mockHttpClient.get.mockReturnValue(of({}));
+
+    const res = await service.getUploadUrl('fileNumber', null);
+    expect(service).toBeTruthy();
```
Comment (trslater) [inline portal-frontend/src/app/services/document/document.service.spec.ts:52]: Won't this always be true? Also, doesn't test whether GET request is made.

Comment (dhaselhan) [reply portal-frontend/src/app/services/document/document.service.spec.ts]: Woops! Yep, looks like I copied that from the default unit test and forgot to change it to expect get. Fixed.

**Review by trslater — APPROVED** [review]
LGTM 🚀

**Pull request merged**